lib/matplotlib/pyplot.py

```diff
@@ -2541,16 +2560,20 @@ def acorr(x, hold=None, data=None, **kwargs):
 def angle_spectrum(x, Fs=None, Fc=None, window=None, pad_to=None, sides=None,
                    hold=None, data=None, **kwargs):
     ax = gca()
-    # allow callers to override the hold state by passing hold=True|False
-    washold = ax.ishold()
+    # Deprecated: allow callers to override the hold state
+    # by passing hold=True|False
+    washold = ax._hold
 
     if hold is not None:
-        ax.hold(hold)
+        ax._hold = hold
+        from matplotlib.cbook import mplDeprecation
+        warnings.warn("The 'hold' kwarg is deprecated since 2.0.",
```

Comment:
Hold is a keyword argument; are you asking me to spell that out, as opposed to using the common abbreviation?  I don't mind doing that, I just want to be clear about what the nit is.